### PR TITLE
I'll be fixing title in many places 

### PR DESCRIPTION
**Pull request by ramico**
sift science  to only sift

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -56,7 +56,7 @@ Please help us improve the plugin by enabling [anonymous statistics and error co
 * 2017-01-17
 * Fix: Can't back-fill more than one order per user
 * Fix: Transaction details sent multiple times
-* Fix: Correct the mapping of WooCommerce order status to Sift Science transaction status
+* Fix: Correct the mapping of WooCommerce order status to Sift  transaction status
```
Comment (nabsul) [inline readme.txt:59]: In the past the name was Sift Science. Let's keep the changelog messages using the old name.

**Comment by ramico** [comment]
not worth revert just going to that location and changing it 
ps:revert miss up things -my party- so i'll redo this one manually

File: readme.txt [hunk 2]
```diff
@@ -75,7 +75,7 @@ Please help us improve the plugin by enabling [anonymous statistics and error co
 
 = 0.4.0 Beta =
 * 2016-11-13
-* Plug in Sift Science $transaction event
```
Comment (nabsul) [inline readme.txt:78]: Please undo the change here.

File: readme.txt [hunk 3]
```diff
@@ -93,7 +93,7 @@ Please help us improve the plugin by enabling [anonymous statistics and error co
 * 2016-10-09
 * Implement a logout method (not plugged into wpcom though)
 * Improve webpack to make a smaller React App
-* Add prefix to user_name field (support multiple stores for one Sift Science account)
```
Comment (nabsul) [inline readme.txt:96]: Please undo the change here.

File: readme.txt [hunk 4]
```diff
@@ -108,7 +108,7 @@ Please help us improve the plugin by enabling [anonymous statistics and error co
 
 = 0.3.3 Beta =
 * Fixed open-in-siftscience action
-* Switched to latest Sift Science API endpoints
```
Comment (nabsul) [inline readme.txt:111]: Please undo the change here.
